lib/common/entropy_common.c

```diff
@@ -200,7 +200,7 @@ BMI2_TARGET_ATTRIBUTE static size_t FSE_readNCount_body_bmi2(
         short* normalizedCounter, unsigned* maxSVPtr, unsigned* tableLogPtr,
         const void* headerBuffer, size_t hbSize)
 {
-    return FSE_readNCount_body(normalizedCounter, maxSVPtr, tableLogPtr, headerBuffer, hbSize);
+    return FSE_readNCount_body_bmi2(normalizedCounter, maxSVPtr, tableLogPtr, headerBuffer, hbSize);
```

Comment:
The original code here is actually correct :) `BMI2_TARGET_ATTRIBUTE` tells the compiler to use BMI2 instructions when compiling this function. But the source code of the function doesn't change.

The reason you're seeing failures in CI is that you're defining `FSE_readNCount_body_bmi2()` in terms of itself, i.e. you have an infinite recursion.